src/app/app.component.ts

```diff
@@ -13,11 +13,9 @@ export class AppComponent implements OnInit {
     console.log('page is fully loaded');
   }
   ngOnInit(): void {
-    this.title = 'Angular CLI Rules';
-    const TIME = 3000;
-    setTimeout(() => (this.title = 'testing'), TIME);
+    //empty
```

Comment:
Thanks for this. 
I was testing something when I wrote those 3 lines. Forgot to delete them
```suggestion
   
```